Sources/Player.swift

```diff
@@ -441,7 +458,7 @@ extension Player {
         let keys: [String] = [PlayerTracksKey, PlayerPlayableKey, PlayerDurationKey]
 
         self._asset.loadValuesAsynchronously(forKeys: keys, completionHandler: { () -> Void in
-            DispatchQueue.main.sync(execute: { () -> Void in
+            DispatchQueue.main.async(execute: { () -> Void in
```

Comment:
The change might be okay, I just have to look at the larger scope and shouldn't comment without looking at the actual PR. ;-)